plugins/modules/system/parted.py

```diff
@@ -557,12 +557,12 @@ def parted(script, device, align):
         align_option = ''
 
     if script and not module.check_mode:
-        command = "%s -s -m %s %s -- %s" % (parted_exec, align_option, device, script)
+        command = [parted_exec, '-s', '-m', align_option, device, '--'] + script
```

Comment:
I found another bug by simply trying to run this module. (Did really nobody do that?!) `align_option` should be a list, not a string. It should either be `['-a', align]` or `[]`, and this line should be
```suggestion
        command = [parted_exec, '-s', '-m'] + align_option + [device, '--'] + script
```